kubernetes/structures_certificate_signing_request_v1.go

```diff
@@ -15,6 +16,9 @@ func expandCertificateSigningRequestV1Spec(csr []interface{}) *certificates.Cert
 		return obj
 	}
 	in := csr[0].(map[string]interface{})
+	if v, ok := in["expiration_seconds"].(int); ok && v >= 600 {
+		obj.ExpirationSeconds = ptr.To(int32(v))
```

Comment:
According to [this](https://kubernetes.io/docs/reference/kubernetes-api/authentication-resources/certificate-signing-request-v1/#CertificateSigningRequestSpec) this is an int32, so I used that here instead of int64. I hope that's correct?

Reply:
This is fine, int32 casting is used in other areas of the provider such as here:
https://github.com/hashicorp/terraform-provider-kubernetes/blob/8ded5dc5338b28f49854d1192f09776ced73e91c/kubernetes/structure_cron_job_v1.go#L90-L97